src/promptflow-devkit/promptflow/batch/_base_executor_proxy.py

```diff
@@ -36,6 +36,19 @@
 
 
 class AbstractExecutorProxy:
+
+    def __init__(self):
+        self._should_apply_inputs_mapping = True
+        self._allow_aggregation = True
```

Comment:
maybe better to default to false?

Reply:
@D-W- Note here for Flex Flow which could be false or true depends on the flow?

Reply:
Do we want to have ability to disable aggregation for chat group batch run?